frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -235,8 +236,22 @@ class Sidebar extends Component<Props> {
                         </div>
                     }
                     { accounts && accounts.map(this.getAccountLink) }
+                    
                     <div className="sidebarHeaderContainer end">
-                        <span className="sidebarHeader">{t('sidebar.settings')}</span>
+                        {/*<span className="sidebarHeader">{t('sidebar.settings')}</span>*/}
+                    </div>
+                    <div className="sidebarItem">
+                        <Link
+                            activeClassName="sidebar-active"
+                            href="/exchanges"
+                        >
+                            <div className="single">
+                                <img draggable={false} className="sidebar_settings" src={diamond} alt={t('sidebar.bitboxBase')} />
```

Comment:
shouldn't draggable be false by default?
also, why `bitboxBase`?

Reply:
nope draggable is true by default https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable

but bitboxbase is a mistake.. will fix

Reply:
fixed in a4a64e47323031e31d608dcb470b7a7256a2cfbd